internal/preferences.go

```diff
@@ -41,7 +41,10 @@ func (p *InMemoryPreferences) BoolListWithFallback(key string, fallback []bool)
 	if !ok {
 		return fallback
 	}
-
+	_, ok1 := value.([]bool)
```

Comment:
There is no need to call this `ok1`. Just change it like this:
```go
	_, ok = value.([]bool)
```

Reply:
done